bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/XmlThingTypeProvider.java

```diff
@@ -299,13 +101,41 @@ public void unsetConfigDescriptionProvider(ConfigDescriptionProvider configDescr
         this.configDescriptionProvider = null;
     }
 
-    @Reference(target = "(esh.scope=core.xml)")
+    @Reference(target = "(esh.scope=core.xml.channels)")
```

Comment:
This should simply create a "target" property in the generated DS XML and these properties are already supported in DS1.1/OSGi4.2, see specification chapter 112.3.4.